### PR TITLE
Update UG/DG

### PR DESCRIPTION
**Pull request by Yuvaraj0702**

### CODE REVIEW

File: docs/diagrams/FindCommandActivityDiagram.puml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+@startuml
+'https://plantuml.com/activity-diagram-beta
+Title:Finding students
+start
+:Recieves a find command;
```
Comment (jorrdansoh) [inline docs/diagrams/FindCommandActivityDiagram.puml:5]: There is a typo here

File: docs/diagrams/FindSequenceDiagram.puml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+activate CommandResult
+
+CommandResult -> FindCommand : r
+deactivate CommandResult
+FindCommand --> LogicManager : r
+deactivate FindCommand
+
+
+[<--LogicManager : r
+deactivate LogicManager
```
Comment (jorrdansoh) [inline docs/diagrams/FindSequenceDiagram.puml:60]: There is a bug in the diagram: all arrows to the right should be solid, while all return arrows should be dashed. Also, showing the object the method is called on, e.g. `fs.setPredicate(p)` is actually not necessary, so you can considering removing them to make the diagram cleaner

**Review by jorrdansoh — CHANGES_REQUESTED** [review]
Do look at the suggested changes before merging 🙂

File: docs/diagrams/FindCommandActivityDiagram.puml (hunk 1 above)
Comment (Yuvaraj0702) [reply docs/diagrams/FindCommandActivityDiagram.puml]: changed

File: docs/diagrams/FindSequenceDiagram.puml (hunk 2 above)
Comment (Yuvaraj0702) [reply docs/diagrams/FindSequenceDiagram.puml]: changed

File: docs/UserGuide.md [hunk 3]
```diff
@@ -323,6 +323,22 @@ If your changes to the data file makes its format invalid, FRIDAY will ignore al
...
+**A**: Yes Friday is still for you. Why you may ask? The commands are still very short and even if you are a slow typer <br>
+       you will be saving time as compared to pen and paper or an excel sheet. <br>
+       Furthermore, FRIDAY is more aesthetically pleasing and has all the necessary features easily accessible at your fingertips.
+
+**Q**: What are the additional features coming to FRIDAY?<br>
+**A**: FRIDAY is always expanding to serve you better. In the near future you can expect to see. 
+       1. Shared databases 
+       2. Larger storage limits 
+       3. Cloud compatibility 
+       4. Easy import and export of data
```
Comment (jorrdansoh) [inline docs/UserGuide.md:340]: Great job adding more QnA!

File: docs/UserGuide.md [hunk 4]
```diff
@@ -346,3 +362,31 @@ If your changes to the data file makes its format invalid, FRIDAY will ignore al
...
+
+9. Midterm: Midterm examination a major assessment that occurs halfway through the CS1101S module.
+
+10. Practical: Practical examination is a major non-paper assessment that occurs at the end of the CS1101S module.
+
+11. Final: Final examination is a major paper assessment that occurs at the end of the CS1101S module.
+
+12. Mastery Check: A pass/fail for of assessment on students to assess their understanding of the concepts taught. There are 2 Mastery checks per semester.
+
+13. Alias: A nickname or an alternate name for a command that you can set.
```
Comment (jorrdansoh) [inline docs/UserGuide.md:392]: Good job adding a Glossary despite not being a requirement :+1:

File: docs/diagrams/FindSequenceDiagram.puml [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+LogicManager -> FindCommand : execute(m)
+activate FindCommand
+
+FindCommand [MODEL_COLOR]-> Model : updateFilteredStudentList(p)
+activate Model
+
+Model [MODEL_COLOR]-> FilteredList : setPredicate(p)
+activate FilteredList
+
```
Comment (jorrdansoh) [inline docs/diagrams/FindSequenceDiagram.puml:44]: Good job implementing the suggested changes! The sequence diagram looks good now

**Review by jorrdansoh — APPROVED** [review]
Thanks for the contribution! LGTM :+1:

**Pull request merged**